pkgs/development/python-modules/aiolifx/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv,
+  fetchPypi,
+  buildPythonPackage,
+  ifaddr,
+  bitstring }:
+
+buildPythonPackage rec {
+  pname = "aiolifx";
+  version = "0.6.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cf53c9faea6eee25a466e73eef1753b82a75c7497648149c19c15342df2678f2";
+  };
+
+  // tests are not implemented :
+  // nix_run_setup: error: unrecognized arguments: test
+  doCheck = false;
+
+  nativeBuildInputs = [ ];
+  propagatedBuildInputs = [ bitstring ifaddr ];
+
+  meta = with stdenv.lib; {
+    homepage = "http://github.com/frawau/aiolifx";
+    license = licenses.mit;
+    description = "API for local communication with LIFX devices over a LAN with asyncio.";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

Reply:
should I add myself to the maintainers in a separate (first) commit ?

Reply:
Yes, please.